.github/workflows/tests.yaml

```diff
@@ -20,11 +20,8 @@ jobs:
           cmake -S . -B build -DATSDK_BUILD_TESTS="unit" -DCMAKE_BUILD_TYPE="Debug"
           cmake --build build --target all
 
-      - name: atchops Unit CTest
-        run: ctest --test-dir build/packages/atchops/tests --output-on-failure --timeout 5 -VV
-
-      - name: atclient Unit CTest
-        run: ctest --test-dir build/packages/atclient/tests --output-on-failure --timeout 5 -VV
+      - name: Run tests
+        run: ctest --test-dir build --output-on-failure --timeout 5 -VV
```

Comment:
Running ctest from the root of the build runs every **enabled** test in the CMake build tree